.github/ci.sh

```diff
@@ -128,24 +73,10 @@ build() {
   fi
 }
 
-build_abc() {
-  arch=X86_64
-  case "$RUNNER_OS" in
-    Linux) os="Linux" ;;
-    macOS) os="OSX" ;;
-    Windows) os="Windows" ;;
-  esac
-  (cd deps/abcBridge &&
-    scripts/build-abc.sh $arch $os &&
-    cp abc-build/abc $BIN/abc)
-  output path $BIN/abc
-}
-
 install_system_deps() {
-  install_z3 &
-  install_cvc4 &
-  install_yices &
-  wait
+  (cd $BIN && curl -o bins.zip -sL "https://github.com/GaloisInc/what4-solvers/releases/download/$SOLVER_PKG_VERSION/$BIN_ZIP_FILE" && unzip -o bins.zip && rm bins.zip)
+  chmod +x $BIN/*
+  cp $BIN/yices_smt2$EXT $BIN/yices-smt2$EXT
```

Comment:
Oh, I see now—this is changing from `yices_smt2` (with an underscore) to `yices-smt2` (with a hyphen). Is this something that should be done on `what4-solvers`' end, I wonder?

Reply:
Yeah, it ultimately should be done over there. But I'm waiting for a time when there are other changes that need to be made over there, too, because the build process takes a while.